### DISCUSSION TITLE
ESP32-CAM draws current after initialization

### DISCUSSION THREAD

**Discussion by ossapandra** [arendst/Tasmota#14732] [Q&A] [answered]
Hello, when I power-on the ESP32-CAM running Tasmota, it draws about 50mA current and connected to WiFi and idling. But when I start the Camera, for example using the command "wcinit", the current draw increases to ~110mA, even when the camera/device is not accessed and supposedly idling. I get the impression that Tasmota is constantly capturing pictures in a buffer to have it ready in case someone might request it. Can it be made so that the device sleeps until I actively request a picture, and after that was captured and send, the device goes back to sleep again? Thank you.

**Comment by barbudor** [comment, marked as answer]
This is how it works
If you want to propose an alternative behavior, a PR may be welcomed
